api/spec/v1.yaml

```
@@ -1903,6 +1903,11 @@ components:
             Retrieving the other signers is currently not supported by this API.
           # NOTE: To expose the other signers, use the transaction_signers table.
           example: oasis1qz670t637yyxshnlxhjj5074wgwl94d0x5x69zqd
+        sender_eth_0:
```

nit: `eth` is the modifier of `sender_0`, so let's either prefix or postfix it, not stick it in the middle of the name. Realistically, we should postfix it for consistency with other fields.

```suggestion
        sender_0_eth:
```

---

Ah fair. I like the index at the end, but you are right, `sender_0_eth` is oddly more consistent.